test/mocks/get-current-price.ts

```diff
@@ -21,3 +24,12 @@ export const getCurrentUsdCentPrice = ({
     currency: currency as DisplayCurrency,
   })
 }
+
+export const getCurrentPriceAsPriceRatio = async ({
+  currency,
+}: GetCurrentSatPriceArgs): Promise<PriceRatio | PriceServiceError> => {
+  const price = await getCurrentSatPrice({ currency })
+  if (price instanceof Error) return price
+
+  return toPriceRatio(price.price * CENTS_PER_USD)
+}
```

Comment:
were you getting an error without this? in theory if code did not change this is not necessary

Reply:
Yup tests fail with "undefined" function for that method